site/troubleshooting.md

```diff
@@ -161,8 +161,9 @@ The columns are as follows:
     * `pending` - TCP connection up, waiting for confirmation of
        UDP heartbeat
     * `established` - TCP connection and corresponding UDP path are up
- * Info - the remote peer name and nickname for (un)established
-   connections, the failure reason for failed and retrying connection
+ * Info - the traffic transport method, remote peer name and nickname
+   for (un)established connections, the failure reason for failed and
+   retrying connection
```

Comment:
I know this was wrong before, but there is no such thing as "unestablished". Should say "pending and established". Also, I reckon "failed and retrying" should come before "pending and established", to match the order in Status.